sys/net/gnrc/transport_layer/tcp/gnrc_tcp_fsm.c

```diff
@@ -617,33 +617,33 @@ static int _fsm_rcvd_pkt(gnrc_tcp_tcb_t* tcb, gnrc_pktsnip_t *in_pkt, bool *noti
             /* ... send RST, seq_no = snd_nxt, ack_no = rcv_nxt */
             _pkt_build(tcb, &out_pkt, &seq_con, MSK_RST, tcb->snd_nxt, tcb->rcv_nxt, NULL, 0);
             _pkt_send(tcb, out_pkt, seq_con, false);
-            _transition_to(tcb, GNRC_TCP_FSM_STATE_CLOSED, notify_owner);
+            _transition_to(tcb, FSM_STATE_CLOSED, notify_owner);
             return 0;
         }
         /* 4) Check ACK */
         if (!(ctl & MSK_ACK)) {
             return 0;
         }
         else {
-            if (tcb->state == GNRC_TCP_FSM_STATE_SYN_RCVD) {
+            if (tcb->state == FSM_STATE_SYN_RCVD) {
                 if (LSS_32_BIT(tcb->snd_una, seg_ack) && LEQ_32_BIT(seg_ack, tcb->snd_nxt)) {
                     tcb->snd_wnd = seg_wnd;
                     tcb->snd_wl1 = seg_seq;
                     tcb->snd_wl2 = seg_ack;
-                    _transition_to(tcb, GNRC_TCP_FSM_STATE_ESTABLISHED, notify_owner);
+                    _transition_to(tcb, FSM_STATE_ESTABLISHED, notify_owner);
                 }
                 else {
                     _pkt_build(tcb, &out_pkt, &seq_con, MSK_RST, seg_ack, 0, NULL, 0);
                     _pkt_send(tcb, out_pkt, seq_con, false);
                 }
             }
             /* Acknowledgment processing */
-            if (tcb->state == GNRC_TCP_FSM_STATE_ESTABLISHED
-            || tcb->state == GNRC_TCP_FSM_STATE_FIN_WAIT_1
-            || tcb->state == GNRC_TCP_FSM_STATE_FIN_WAIT_2
-            || tcb->state == GNRC_TCP_FSM_STATE_CLOSE_WAIT
-            || tcb->state == GNRC_TCP_FSM_STATE_CLOSING
-            || tcb->state == GNRC_TCP_FSM_STATE_LAST_ACK
+            if (tcb->state == FSM_STATE_ESTABLISHED
```

Comment:
I think was taken directly from the TCPs RFCs "event processing section". I'll take a look into it.

About the coding style issues. I'll do that.